_targets.R

```diff
@@ -0,0 +1,100 @@
+# Targets pipeline for `baselinenowcast` analysis paper.
+# This script defines the analysis pipeline used to evaluate the performance
+# of the `baselinenowcast` R package, which is intended to be used as a
+# baseline method for nowcasting right-truncated epidemiological data.
+
+# The performance of the baseline nowcast method will be evaluated as follows:
+# 1. Applied to real-world data from the German Nowcast Hub from 1 Dec 2021 to
+# 1 April 2022, nationally, by age group, and by state.
+# 2. Applied to 4 simulated data scenarios in which we take two age strata, one
+# sparse and one rich from the German Nowcast data, and apply a long and short
+# delay to the data. For each data scenario, we will assess 24 different
+# combinations of model specifications
+# 3. Applied to real-world data from 1. 2013-2014 Measles outbreak in the
+# Netherlands 2. 2023-2024 Norovirus outbreak in the UK and 3. Comparison of
+# performance of the mean/median ensemble in the German Nowcast Hub
+
+library(targets)
+library(crew)
+library(RcppTOML)
+library(readr)
+
+controller <- crew_controller_local(
+  workers = 8,
+  seconds_idle = 600
+)
+
+# Set targets options:
+tar_option_set(
+  packages = c(
+    "tibble", "dplyr", "lubridate",
+    "baselinenowcast",
+    "readr"
+  ),
+  workspace_on_error = TRUE,
+  # Run with a pre-specified crew controller
+  controller = controller,
+  # Setup storage on workers vs on the main node.
+  memory = "transient",
+  storage = "worker",
+  retrieval = "worker",
+  format = "rds", # default storage format
+  error = NULL # tells errored targets to return NULL rather than
+  # have whole pipeline fail
+)
+
+# Swap in with real config, use example for now
```

Comment:
Do we need or want a config here? What is it for?

Reply:
I have had it drilled into me that we always need a config -- I do find it easier to test a small subset of the workflow using it and then only run the full thing when needed

Reply:
Removed the config, though I think there is a chance I might still bring it back if it feels useful for testing smaller runs of the pipeline